rules/fixtures/rules_multiple_groups.yaml

```diff
@@ -0,0 +1,28 @@
+groups:
+  - name: http
+    rules:
+      # independents
+      - record: job:http_requests:rate1m
+        expr: sum by (job)(rate(http_requests_total[1m]))
+      - record: job:http_requests:rate5m
+        expr: sum by (job)(rate(http_requests_total[5m]))
+
+      # dependents
+      - record: job:http_requests:rate15m
+        expr: sum by (job)(rate(http_requests_total[15m]))
+      - record: TooManyHTTPRequests
+        expr: job:http_requests:rate15m > 100
+
+  - name: grpc
```

Comment:
Can we have them spliced in one test?

As in:

1. job:http_requests:rate1m
2. job:http_requests:rate15m
3. job:http_requests:rate5m
4. TooManyHTTPRequests

Reply:
@gouthamve I'm not understanding what you're asking, can you elaborate please?
They're all part of the same group and tested in `TestBoundedRuleEvalConcurrency`.